src/components/region-common.js

```diff
@@ -64,26 +66,38 @@ export const allRegions = [
-  // { value: 'EUROPE-WEST4', label: 'europe-west4 (Netherlands)', locationType: 'region' },
-  // { value: 'EUROPE-WEST6', label: 'europe-west6 (Zurich)', locationType: 'region' },
-  // { value: 'ASIA-EAST1', label: 'asia-east1 (Taiwan)', locationType: 'region' },
-  // { value: 'ASIA-EAST2', label: 'asia-east2 (Hong Kong)', locationType: 'region' },
-  // { value: 'ASIA-NORTHEAST1', label: 'asia-northeast1 (Tokyo)', locationType: 'region' },
-  // { value: 'ASIA-NORTHEAST3', label: 'asia-northeast3 (Seoul)', locationType: 'region' },
-  // { value: 'ASIA-SOUTH1', label: 'asia-south1 (Mumbai)', locationType: 'region' },
-  // { value: 'ASIA-SOUTHEAST1', label: 'asia-southeast1 (Singapore)', locationType: 'region' },
-  // { value: 'ASIA-SOUTHEAST2', label: 'asia-southeast2 (Jakarta)', locationType: 'region' },
-  // { value: 'AUSTRALIA-SOUTHEAST1', label: 'australia-southeast1 (Sydney)', locationType: 'region' }
+  { value: 'US-CENTRAL1', label: 'us-central1 (Iowa)', locationType: locationTypes.region },
+  { value: 'NORTHAMERICA-NORTHEAST1', label: 'northamerica-northeast1 (Montreal)', locationType: locationTypes.region },
+  { value: 'SOUTHAMERICA-EAST1', label: 'southamerica-east1 (Sao Paulo)', locationType: locationTypes.region },
+  { value: 'EUROPE-CENTRAL2', label: 'europe-central2 (Warsaw)', locationType: locationTypes.region },
+  { value: 'EUROPE-NORTH1', label: 'europe-north1 (Finland)', locationType: locationTypes.region },
+  { value: 'EUROPE-WEST1', label: 'europe-west1 (Belgium)', locationType: locationTypes.region },
+  { value: 'EUROPE-WEST2', label: 'europe-west2 (London)', locationType: locationTypes.region },
+  { value: 'EUROPE-WEST3', label: 'europe-west3 (Frankfurt)', locationType: locationTypes.region },
+  { value: 'EUROPE-WEST4', label: 'europe-west4 (Netherlands)', locationType: locationTypes.region },
+  { value: 'EUROPE-WEST6', label: 'europe-west6 (Zurich)', locationType: locationTypes.region },
+  { value: 'ASIA-EAST1', label: 'asia-east1 (Taiwan)', locationType: locationTypes.region },
+  { value: 'ASIA-EAST2', label: 'asia-east2 (Hong Kong)', locationType: locationTypes.region },
+  { value: 'ASIA-NORTHEAST1', label: 'asia-northeast1 (Tokyo)', locationType: locationTypes.region },
+  { value: 'ASIA-NORTHEAST3', label: 'asia-northeast3 (Seoul)', locationType: locationTypes.region },
+  { value: 'ASIA-SOUTH1', label: 'asia-south1 (Mumbai)', locationType: locationTypes.region },
+  { value: 'ASIA-SOUTHEAST1', label: 'asia-southeast1 (Singapore)', locationType: locationTypes.region },
+  { value: 'ASIA-SOUTHEAST2', label: 'asia-southeast2 (Jakarta)', locationType: locationTypes.region },
+  { value: 'AUSTRALIA-SOUTHEAST1', label: 'australia-southeast1 (Sydney)', locationType: locationTypes.region }
 ]
+
+// For current phased release of regionality only supporting US and US-CENTRAL1 buckets.
+export const availableBucketRegions = allRegions.slice(0, 2)
```

Comment:
We typically recommend using Lodash functions over the native functions for the following reasons:

- Lodash will handle the case where you try to run the function on an incorrect type. e.g.: null, undefined, Object  (in the case of slice)
  - {}.slice(2,4) // throws an error
  - _.slice(2, 4, {}) // returns an empty array
- When running through your code, if you pulled data in from an outside source and got unexpected garbage, the application will not have an error thrown, and can continue along with the value / type it was expecting when using lodash
- Lodash/fp functions support auto currying. This is especially useful for code changes later. If someone wants to add functionality, it makes it easy to do something like add the function to a flow
- Lodash/fp will not mutate the array. In the case of native slice, it also returns a new array. But there are other functions that can mutate (e.g. push/pop)
- For consistency within the codebase, consistency with how the code operates, and to prevent mutation we tend to prefer using lodash functions.
-Mixing native and lodash functions can lead to confusion about behavior.

Reply:
Fixed. Thanks! I have trouble remembering which functions are available in lodash, so appreciate the reminders.